metrics/src/main/java/io/opencensus/metrics/Value.java

```diff
@@ -173,10 +174,19 @@ public static DistributionValue create(
         double mean,
         long count,
         double sumOfSquaredDeviations,
-        Range range,
+        @Nullable Range range,
         List<Double> bucketBoundaries,
         List<Bucket> buckets) {
       Utils.checkArgument(count >= 0, "count should be non-negative.");
+      Utils.checkArgument(
+          sumOfSquaredDeviations >= 0, "sum of squared deviations should be non-negative.");
+      if (count == 0) {
+        Utils.checkArgument(range == null, "range should not be present if count is 0.");
+        Utils.checkArgument(
+            sumOfSquaredDeviations == 0, "sum of squared deviations should be 0 if count is 0.");
```

Comment:
I think you could also check that `sum` is zero here.

Reply:
Done.